eva-lib/src/main/java/uk/ac/ebi/eva/lib/filter/VariantEntityRepositoryFilter.java

```diff
@@ -27,13 +27,13 @@
 import java.util.Objects;
 import java.util.stream.Collectors;
 
-public class VariantRepositoryFilter<T> {
+public class VariantEntityRepositoryFilter<T> {
```

Comment:
Question to the quorum: Maybe we should simplify the naming for this filter family (ValueFilter or something smaller)  as this kind of filter could be used in more places or contexts than the VariantEntityRepository. Then we could rename the rest of the filters as MafFilter, PolyphenFilter, etc... Right now I think it's a little verbose. Any thoughts?